utils.py

```diff
@@ -31,12 +29,19 @@ def take_screenshot():
 
 
 def prepare_image(img):
+    import wx
+    wx.App()
```

Comment:
Yeah that seems reasonable. I'm sure there's a few things we can clean up now that a lot of the framework pieces are handled by the environment. Honestly, I haven't focused much on the agent code and have mostly spent my time on the gym components. I haven't put any time into `record.py`. I will update the PR once I get that refactored.

I wonder if we could even get the record stuff working through the gym environment too. I may play with that idea a bit, but don't hold up merging this PR. If I follow-through with that, I'll submit a different one.